data/assets/06_legend.css

```diff
@@ -0,0 +1,30 @@
+@import './01.css';
+
+/* Legend explaining colors; under the connection matrix */
+.chart_legend{
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  font-size: var(--font-size-small);
+  color: var(--theme-1);
+}
+.chart_legend__label{
+  margin: 0 10px 0 30px;
+}
+
+.chart_legend__cell {
+  width: 30px;
+  height: 16px;
+  border-radius: 4px;
+  display: inline-block;
+}
+.chart_legend__cell_good {
+  background: #0f0;
```

Comment:
The legend color has to match the background in cells. There should to be variables defining the "Good", "Warning" and "Error" colors and they must be used everywhere the color is needed.

Also we should match the Kentik UI naming for test health, so it should be "Healthy", "Warning" and "Critical".